### ISSUE TITLE
Export as ics file

### ISSUE THREAD

**Issue by mielvds** [iRail/hyperRail#256]
Nice to have: an export to calendar for every route. Maybe with this script: https://gist.github.com/jakebellacera/635416

**Comment by avdg** [comment]
What about a composer package to solve the problem? I saw something like https://packagist.org/packages/eluceo/ical

**Comment by kidk** [comment]
I want to take some time implementing this, but I could use some feedback. As far as I can see we have two options:
- Adding the composer package mentioned above and generating the ICS file in PHP. This would mean sending all the data through a GET/POST request.
- Adding a Javascript dependency ( https://github.com/nwcell/ics.js ) to generate the file in the browser. This looks promising, but activity on the project is low. 

My feeling is that the GET/POST approach would work best, where I send the information from /route/ call back to the back-end to generate the ICS file.

**Comment by mielvds** [comment]
I agree, that will also allow to archive and cache the ics files if desired.

**Comment by kidk** [comment]
Pull request is live, still a bit WIP, but comments welcome. https://github.com/iRail/hyperRail/pull/260
